src/pages/Groups/components/GroupAdmin.tsx

```diff
@@ -15,16 +15,17 @@ import TextField from 'components/inputs/TextField';
 import SubmitButton from 'components/inputs/SubmitButton';
 import Dialog from 'components/layout/Dialog';
 import { ShowMoreText, ShowMoreTooltip } from 'components/miscellaneous/UserInformation';
+import { ImageUpload } from 'components/inputs/Upload';
 
 export type UpdateGroupModalProps = {
   group: Group;
 };
 
-type FormValues = Pick<Group, 'contact_email' | 'description' | 'fine_info' | 'fines_activated' | 'name' | 'fines_admin'>;
+type FormValues = Pick<Group, 'contact_email' | 'description' | 'fine_info' | 'fines_activated' | 'name' | 'fines_admin' | 'image' | 'image_alt'>;
```

Comment:
Vi bruker ikke `image_alt` så da trenger du den heller ikke i `FormValues`-typen
```suggestion
type FormValues = Pick<Group, 'contact_email' | 'description' | 'fine_info' | 'fines_activated' | 'name' | 'fines_admin' | 'image'>;
```